.github/workflows/main.yml

```diff
@@ -382,6 +382,13 @@ jobs:
                 make delete
             fi
 
+  delete-test-images:
```

Comment:
doing this in another step does not work ok since the checkout is done again -> specific set version is lost

Reply:
yeah better to move the delete images to their own step. Also, we can skip that step until we figure out why the delete image is failing.

Reply:
I'm now passing the version, hopefully that should fix the logic.
Now the deletion is done in another step that should not prevent publication: maybe can merge as is and figure out deletion issues later (that will still make the PR status fail until deletion problem is resolved)

Reply:
Added the step to pass that version in a file named "VERSION" for Makefile logic...